web-ng/src/app/services/observation/observation.service.ts

```diff
@@ -41,35 +52,71 @@ export class ObservationService {
   constructor(
     private dataStore: DataStoreService,
     projectService: ProjectService,
-    featureService: FeatureService
+    featureService: FeatureService,
+    authService: AuthService
   ) {
     this.subscription.add(
       this.selectedObservationId$
         .pipe(
           switchMap(observationId =>
-            projectService
-              .getActiveProject$()
-              .pipe(
-                switchMap(project =>
-                  featureService
-                    .getSelectedFeature$()
-                    .pipe(
-                      switchMap(feature =>
-                        this.dataStore.loadObservation$(
+            projectService.getActiveProject$().pipe(
+              switchMap(project =>
+                featureService.getSelectedFeature$().pipe(
+                  switchMap(feature =>
+                    authService.getUser$().pipe(
+                      switchMap(user => {
+                        if (observationId === ':new') {
+                          return of(
+                            this.createNewObservation(user, project, feature)
+                          );
+                        }
+                        return this.dataStore.loadObservation$(
                           project,
                           feature,
                           observationId
-                        )
-                      )
+                        );
+                      })
                     )
+                  )
                 )
               )
+            )
           )
         )
         .subscribe(o => this.selectedObservation$.next(o))
     );
   }
 
+  createNewObservation(
+    user: User | null | undefined,
+    project: Project,
+    feature: Feature
+  ): Observation | LoadingState {
+    if (user === null || user === undefined) {
+      return LoadingState.LOADING_FAILED;
+    }
+    const form: Form | undefined = project
```

Comment:
Can the type here be inferred instead of declaring it explicitly?

Reply:
ok.. why do you prefer inferred types? aren't declared types more readable and safer?

Reply:
Mainly for consistency; if we do it here we should do it everywhere, but that would make the code less readable overall. Wdyt?

Reply:
i am fine either way. I changed it already so there is no type here